src/components/footer/footer-bar.vue

```diff
@@ -0,0 +1,135 @@
+<script setup lang="ts">
+import ButtonIcon from './button-icon.vue'
+import ButtonLink from './button-link.vue'
+import { useTranslation } from 'i18next-vue'
+import { useAppStore } from '@/stores/app.store'
+const { t, i18next } = useTranslation()
+const { setLayersOpen } = useAppStore()
+</script>
+<template>
+  <footer>
+    <!-- left buttons -->
+    <div
+      class="w-80 flex flex-row justify-start text-primary divide-x divide-gray-400 divide-solid border-x border-gray-400"
+    >
+      <ButtonIcon
+        :label="t('Layers', { ns: 'client' })"
+        icon="layers"
+        @click="() => setLayersOpen(true)"
+      >
+      </ButtonIcon>
+      <!--TODOs in each button when implemented
+        - remove class="text-gray-300"
+        - add click handler that calls setLayersOpen(true) and opens tool 
+      -->
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('My Maps', { ns: 'client' })"
+        icon="mymaps"
+      >
+      </ButtonIcon>
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Infos', { ns: 'client' })"
+        icon="infos"
+      >
+      </ButtonIcon>
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Legends', { ns: 'client' })"
+        icon="legends"
+      >
+      </ButtonIcon>
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Routing', { ns: 'client' })"
+        icon="routing"
+      >
+      </ButtonIcon>
+    </div>
+
+    <!-- center buttons -->
+    <div
+      class="w-64 flex flex-row justify-start text-primary divide-x divide-gray-400 divide-solid border-x border-gray-400"
+    >
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Dessin', { ns: 'client' })"
+        icon="draw"
+      >
+      </ButtonIcon>
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Mesurer', { ns: 'client' })"
+        icon="measure"
+      >
+      </ButtonIcon>
+      <ButtonIcon
+        class="text-gray-300"
+        :label="t('Imprimer', { ns: 'client' })"
+        icon="print"
+      >
+      </ButtonIcon>
+      <ButtonIcon
```

Comment:
True, this might be more elegant template wise, but should include the three button lists to be consistent IMO (which would also need the CSS classes to be in the array). Given the drawback that dynamically generated CSS classes need to be whitelisted in tailwind, I'll keep it as is for now.